backup_and_restore/index.adoc

```diff
@@ -27,3 +26,37 @@ You might run into several situations where {product-title}  does not work as ex
 * You have lost the majority of your control plane hosts, leading to etcd quorum loss.
 
 You can always recover from a disaster situation by xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restoring your cluster to its previous state] using the saved etcd snapshots.
+
+[id="application-backup-restore-operations-overview"]
+== Application backup and restore operations
+
+As a cluster administrator, you can back up and restore applications running on {product-title} by using the OpenShift API for Data Protection (OADP).
+
+OADP backs up and restores Kubernetes resources and internal images, at the granularity of a namespace, by using link:https://velero.io/[Velero 1.7]. OADP backs up and restores persistent volumes (PVs) by using snapshots or Restic. For details, see xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
+
+You can install OADP with xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[default plugins] that support cloud providers, Container Snapshot Interface (CSI) snapshots, and {product-title} resources.
+
+.OADP requirements
+
+OADP has the following requirements:
+
+* You must be logged in as a user with a `cluster-admin` role.
+* You must have object storage for storing backups:
```

Comment:
I think the language here should clarify what this list does. Right now, I can't answer these questions:
Do we need ALL the items here? 
Can I use only ONE item?
Can I use another object storage service, or must I use one from this list?

Reply:
I will add "such as one of the following examples" or similar.

The last item says "S3-compatible", so, yes, you can use any object storage as long as it supports S3. I think that covers all the possibilities.